ui/src/app/cohort-search/overview/overview.component.tsx

```diff
@@ -260,6 +270,10 @@ export const ListOverview = withCurrentWorkspace()(
         .findDemoChartInfo(+cdrVersionId, genderOrSexType.toString(), ageType.toString(), request, {signal: this.aborter.signal});
     }
 
+    startGenomeExtractionJob() {
```

Comment:
dataset builder seems a more likely destination for this UI code - but I know there is some pending discussion with Shimon on this